.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release @dev tag to npm
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
It will trigger on label, right? So if I want to re-create the release, I need to remove the label and add it again, OR add any extra label after the "release dev tag" label?

I think its fine for starters, but we should likely replace it with "comment" event, to have more control ;)

Reply:
I just tested: 
1. if you have `release dev tag` and you add new label - workflow will be triggered (as you still have trigger label label)
2. if you want trigger release again - remove and add label again